support/apache-release/prepare_rc.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# 1. Tags the HEAD on the 'master' branch with 
+#    'myriad-<version>-incubating-rc<RC#>'.
+# 2. Creates release artifacts and signs them with GPG key.
+# 3. Uploads the release artifacts to "dev" SVN.
+
+test ${#} -eq 2 || \
+  { echo "Usage: `basename ${0}` [myriad version] [rc #]"; exit 1; }
+
+VERSION=${1}
+RC=${2}
+
+TAG="myriad-${VERSION}-incubating-rc${RC}"
+GIT_URL="https://git-wip-us.apache.org/repos/asf/incubator-myriad.git"
+WORK_DIR="."
+
+echo "Preparing a release candidate ${TAG}.."
+
+TEMP="temp"
+# clone the repo to "temp"
+git clone $GIT_URL ${TEMP}|| \
+  { echo "Failed to clone Myriad from ${GIT_URL} into ${TEMP}"; exit 1; }
+
+# perform a build
+pushd ${TEMP}
+./gradlew build || \
+  { echo "Failed to build Myriad."; exit 1; }
+
+echo "Creating git tag ${TAG} and pushing to ${GIT_URL} ...${NORMAL}"
+git tag ${TAG}
+git push ${GIT_URL} refs/tags/${TAG}
+
+popd # temp
+
+# make a fresh clone that syncs to ${TAG}
+git clone $GIT_URL ${TAG} --branch ${TAG} || \
+  { echo "Failed to clone Myriad from ${GIT_URL}"; exit 1; }
+
+# Remove files/folders that shouldn't be included in the release artifacts
+rm -rf ${TAG}/website
+rm -rf ${TAG}/support
+rm -rf ${TAG}/gradlew.bat
+rm -rf ${TAG}/gradlew
+rm -rf ${TAG}/gradle/wrapper
```

Comment:
I'm a little curious why we need to remove gradlew (I know it's apache related), I noticed Kafka leaves the gradlew script in their source tars.  It would be nice to leave gradlew for those w/o gradle installed.  We should get clarification